Include/UrlLib/UrlLib.h

```diff
@@ -49,6 +51,12 @@ namespace UrlLib
 
         arcana::task<void, std::exception_ptr> SendAsync();
 
+        void SetRequestBody(std::string requestBody);
+
+        void SetRequestHeader(std::string name, std::string value);
+
+        std::unordered_map<std::string, std::string> GetAllResponseHeaders() const;
+        
```

Comment:
Maybe put all the header-related functions together?

Reply:
I didn't see a change here. Did I miss something? `GetResponseHeader` is sitting by itself.

Reply:
Oh whoops, think I accidentally resolved the conversation and missed it